src/styles/ErrorPages.scss

```diff
@@ -0,0 +1,17 @@
+.error-page-container {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+  width: 100%;
+  text-align: center;
+  margin: 10% 0;
+  min-height: 200px;
+  align-items: center;
+
+  &__art-work {
+    width: 400px;
+    height: 400px;
```

Comment:
I wonder if to make this responsive, we could add a container around the SVG which uses `vh` as unit (the svg being 100% width and height inside)?

Reply:
It is not not the same width and height inside. I guess that is why this is needed, so that all the illustrations look the same for regarding their position in the page

Reply:
Yes, but we don't really care about the width, only the height should be consistent. I think we can make the width scale based on a fixed height.

Reply:
Changed this to `rem` just so pixel values aren't used, but the simple `ViewPort` approach didn't work. I need to work a little bit more on this to make this responsive.